test/test_um2netcdf.py

```diff
@@ -395,7 +403,12 @@ def __init__(self, item_code, var_name=None, attributes=None, units=None):
         self.units = None or units
         self.standard_name = None
         self.long_name = None
-        self.coord = {}
+
+    def coord(self):
```

Comment:
`coord()` needs a placeholder arg for the coord lookup name. Since we're ignoring it here, the signature can be `def coord(self, _):`, then `DummyCube` & `DummyCubeWithCoords` have the same method.

I suspect `DummyCube` & `DummyCubeWithCoords` can be merged in in a future refactor too.

Reply:
I did a quick experiment to check merging `DummyCube` & `DummyCubeWithCoords` here:
https://github.com/ACCESS-NRI/um2nc-standalone/commit/6cf2e325787d270dccb15292729adfd3d5dd9d78

Merging the classes was straightforward & the tests still passed. I'd defer the real refactor until outstanding PRs are merged, then the change can be assessed against all the affected tests...

Reply:
Added this in!